main.go

```diff
@@ -1395,7 +1440,7 @@ func computePkgCache(fsCache *cache.Cache, lpkg *listedPackage, pkg *types.Packa
 			if err != nil {
 				return err
 			}
-			computedImp, err := computePkgCache(fsCache, lpkg, pkg, files, info)
+			computedImp, err := computePkgCache(fsCache, lpkg, pkg, files, info, ssaPkg)
```

Comment:
this seems wrong - you're reusing the current package's ssaPkg, not the imported package's. I imagine this should be `nil` instead. Maybe we need better tests for the "garble cache miss" edge case.

Reply:
Yes, that's a good point. It should be nil